docs/kb/semgrep-ci/bitbucket-triggering-events-to-jenkins.md

```diff
@@ -0,0 +1,38 @@
+---
+tags:
+  - Bitbucket
+  - Jenkins
+  - webhooks
+description: How to trigger events from the Bitbucket server to a Jenkins instance
+---
+# Triggering events from Bitbucket to Jenkins
+
+## Motivation
+
+A common scenario for many organizations is having:
```

Comment:
```suggestion
This guide walks you through connecting your Bitbucket source code manager (SCM) with Jenkins, a CI/CD environment, through the use of **webhooks**. By connecting Bitbucket with Jenkins, you are able to customize jobs, such as your Semgrep job, based on push or pull events from Bitbucket.
```

- Trimmed some words and applied the [inverted pyramid](https://www.nngroup.com/articles/inverted-pyramid/).
- This method supports all Bitbucket plans, so there's no need to specify.